lib/iframe-browser.js

```diff
@@ -11,7 +11,9 @@ define(function (require, exports, module) {
         HORIZONTAL_ORIENTATION  = 1;
     // by default we use vertical orientation
     var _orientation = VERTICAL_ORIENTATION;
-
+    // Window object reference
+    var windowObjectRef = null;
```

Comment:
Also, you don't need to set this to `null`, it will be `undefined` automatically, and below you can just check for `!detachedWindow` instead of doing `=== null`